### PR TITLE
Security gas-mask hacking can now backfire.

### PR DESCRIPTION
**Pull request by lordpidey**
There is a 20% chance when hacking a sec-mask that it will now give very friendly responses in a high pitched robotic voice instead of aggressive ones.

### CODE REVIEW

**Comment by Anonus** [comment]
I want a way to manually trigger this so you can be NICE COP and the other guy can be MAD COP

File: code/modules/clothing/masks/hailer.dm [hunk 1]
```diff
@@ -164,6 +176,24 @@
...
+			if(21)
+				phrase_text = "When you are finished with your current activity, please follow me to the brig."
+				phrase_sound = "whenfinish"
+			if(22)
+				phrase_text = "The ground wants to give you a hug, it would be rude to not accept."
+				phrase_sound = "groundhug"
+			if(23)
+				phrase_text = "We're police officers! We're not trained to handle this kind of violence!"
+				phrase_sound = "nottrain"
+			if(23)
```
Comment (GinjaNinja32) [inline code/modules/clothing/masks/hailer.dm]: you have a duplicate 23 here

**Comment by Shapsy** [comment]
Hilarious :+1::+1::+1::+1:

**Comment by PKPenguin321** [comment]
here are the new sounds all compacted into a webm: http://puu.sh/lzSAy/307046854d.webm

**Comment by Aranclanos** [comment]
>almost one extra mb of even more secmask sounds, in this case the gimmick of a gimmick

**Comment by Niknakflak** [comment]
These sounds aren't that good honestly.

**Comment by lordpidey** [comment]
Yeah, listening to the sounds again, they suck.  I'm gonna close this until I have better sounds made.

**Pull request closed without merging**